### PR TITLE
Sort meetings in descending order of dates

### PR DESCRIPTION
**Pull request by mugiwarafx**

### CODE REVIEW

File: static/jscn/fragment.js [hunk 1]
```diff
@@ -59,22 +59,32 @@ async function populate_datetxt_list(channel) {
...
+
+        const dataSorted = Object.entries(data);
+        dataSorted.reverse();
+
+        for (let i = 0; i < dataSorted.length; i++) {
+
+            const element = dataSorted[i];
+            const date = element[0];
+            const url = element[1];
+
```
Comment (t0xic0der) [inline static/jscn/fragment.js:71]: Could you please do away with excess in-line spacings and set the indentation right?

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Could you also sort multiple meetings on the same day?

For instance,

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Still not quite there - note the Fedora Project Management meetings in the attached screenshot. I see a 20:00 meeting getting listed after the one at 13:11 even though the former happened after the latter.

**Comment by mugiwarafx** [comment]
Hi t0xic0der 🙂️

Seems like some kind of dark magic is going on here... Let me check my DADA book.

Just kidding! It looks like your browser is serving the old js file... could that be the case?

It is apparently working correctly on my machine:

Screenshot from 2021-09-04 15-42-15

**Comment by t0xic0der** [comment]
Yep, you are right.

After I spammed `Ctrl+Shift+R` multiple times, it loaded up the way it was supposed to.

**Comment by t0xic0der** [comment]
@thunderbirdtr @lilyx13 @nasirhm @ramyaparimi - Can I have a couple of +1s from your end too before I approve of this merge?

**Review by t0xic0der — APPROVED** [review]

**Pull request merged**